supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -613,6 +624,28 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*: xref:target[target]
 
+[discrete]
+[[source-rpm]]
+==== image:images/yes.png[yes] Source RPM (noun)
+*Description*: _Source RPM_, or SRPM, is an RPM package that contains the complete source code, including any patches, and a SPEC file that describes how to build the source code into a binary RPM.
+
+*Use it*: yes
+
+*Incorrect forms*: srpm
+
+*See also*: xref:spec-file[spec file], xref:rpm[RPM]
+
+// Fuse: Added "In Red Hat Fuse," and removed "In Fuse tooling,"
+[discrete]
+[[source-tab]]
+==== image:images/yes.png[yes] Source tab (noun)
```

Comment:
I'm guessing that this was an unintentional inclusion. This was removed purposefully by another person in #231.